drivers/scheduler/k8s/k8s.go

```diff
@@ -5270,6 +5422,21 @@ func MakePVCFromSnapshot(ns string, name string, sourceSnapshotName string, stor
 	return PVCSpec
 }
 
+// MakePVCByCloning takes namespace, name, source snapshot name and storageclass as parameter and returns a PersistentVolumeClaim spec for PVC creation
+func MakePVCByCloning(size resource.Quantity, ns string, name string, sourcePVCName string, storageClass string) *v1.PersistentVolumeClaim {
```

Comment:
Maybe call this `GeneratePVCCloneSpec`? "Make" makes it sound like it's actually running the create.